src/foreground-plugin.ts

```diff
@@ -0,0 +1,203 @@
+import { CallContext, RESET, GET_BLOCK, BEGIN, END, ENV, STORE } from './call-context.ts';
+import { PluginOutput, type InternalConfig } from './interfaces.ts';
+import { loadWasi } from 'js-sdk:wasi';
+
+export const DYLIBSO_ENV = 'env';
```

Comment:
@chrisdickinson what do you think about naming this, `EXTISM_ENV`?

Reply:
Ooh, good catch!